kpops/components/streams_bootstrap_v3/streams/streams_app.py

```diff
@@ -67,7 +66,7 @@ class StreamsAppV3(StreamsBootstrapV3):
         description=describe_attr("values", __doc__),
     )
 
-    version: str | None = STREAMS_BOOTSTRAP_V3
+    version: str | None = STREAMS_BOOTSTRAP_VERSION
```

Comment:
IMO we can also remove these fields from streams_app and producer_app. WDYT?

Reply:
yes 👍🏻